voctocore/tests/helper/voctomix_test.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
         lib.config.Config.resetToDefaults()
 
     def assertContainsIgnoringWhitespace(self, text, search):
-        regex = search.replace(" ", "\s*")
+        regex = search.replace(" ", "\\s*")
```

Comment:
Reading https://docs.python.org/3.1/library/re.html this could probably also be written als `r"\s"`, because the prefixing `r` denotes a raw string (and not a regex, as one might guess):
> The solution is to use Python’s raw string notation for regular expression patterns; backslashes are not handled in any special way in a string literal prefixed with 'r'. So r"\n" is a two-character string containing '\' and 'n', while "\n" is a one-character string containing a newline. Usually patterns will be expressed in Python code using this raw string notation.